### PR TITLE
Added invoke test_harvesters to test all harvester tests

### PR DESCRIPTION
**Pull request by PatrickEGorman**

### CODE REVIEW

**Comment by PatrickEGorman** [comment]
Related to https://github.com/fabianvf/scrapi/issues/208

**Comment by fabianvf** [comment]
I think this has a similar functionality to the existing ```invoke test``` task. The harvester test generation referred to in #208 involves generating a request/response yaml file inside of the ```tests/vcr``` directory for a given harvester class.

**Comment by PatrickEGorman** [comment]
Okay.  Regardless of how practical it is, I learned some about tasks and tests from it so no big loss.

File: tasks.py [hunk 1]
```diff
@@ -88,6 +89,22 @@ def test(cov=True, verbose=False, debug=False):
...
+    """
+    cmd = 'py.test tests'
+    args = ''
+    if verbose:
+        args += ' -v'
+    if debug:
+        args += ' -s'
+    if cov:
+        args += ' --cov-report term-missing --cov-config .coveragerc --cov scrapi'
+    map(lambda x: run('py.test tests/' + x + args) if('harvester' in x) else(), os.listdir('tests'))
```
Comment (fabianvf) [inline tasks.py]: As an additional, stylistic note, I think maps are usually best used in functions where you don't have side-effects, but have a transformation that you want to perform on each element of the list. For example 
```
map(str, [1, 2, 3, 4])
```
Takes a list of integers and returns a list of strings. As another example, say you wanted only the first character of a list of strings, you could do:
```
map(lambda x: x[:1], ['hector', 'earl', 'lenny', 'larry', 'oswald'])
```
But when you have side-effects like the above function, and the return value of the function you are mapping isn't important to you, a for loop is probably the better choice.

**Comment by fabianvf** [comment]
Yeah, absolutely. If you want me to give you a quick walkthrough of issue #208, I would be happy to. It's a pretty tricky little problem/

**Comment by PatrickEGorman** [comment]
In a little I'd be glad.  I have to finish something up real quick

**Pull request closed without merging**